src/helpers/profile.ts

```diff
@@ -67,3 +67,28 @@ export async function getProfiles(addresses) {
     })
   );
 }
+
+export async function ensTextRecord(ens: string, record: string) {
```

Comment:
Can you move this function to `OptionApproval.vue` for now? It's only used there and hardcoded to mainnet. Maybe we can consider changing our [`getSpaceUri`](https://github.com/snapshot-labs/snapshot.js/blob/f66f47c9211129c262c9c012774de8beef5e9bb8/src/utils.ts#L168) function into a generic `getEnsTextRecord` function and then use that.

Reply:
I was looking for a function like that one! I applied your suggestion in [this PR](https://github.com/snapshot-labs/snapshot-plugins/issues/56). 👍🏻